stitching-directives-with-code-first/README.md

```diff
@@ -0,0 +1,246 @@
+# Chapter X – Stitching directives with Code-First Schemas
```

Comment:
Chapter numbers have been removed! The book outline got broken down into some higher-level sections, so this now has a home under https://github.com/gmac/schema-stitching-handbook#other-integrations.

I'd suggest naming the package `code-first-schemas` now that that's more or less the unique focus of the chapter.

Otherwise, that's all my planned chapters!